flytekit/core/base_task.py

```diff
@@ -304,16 +311,21 @@ def __init__(
         name: str,
         task_config: T,
         interface: Optional[Interface] = None,
-        environment=None,
-        task_type_version=0,
+        environment: Optional[Dict[str, str]] = None,
```

Comment:
why are we removing `task_type_version`?

Reply:
not removing, i am keeping it so that, if something is declared in the base, it does not need to repeated at every class layer - **kwargs should take care of it?